.travis.yml

```diff
@@ -2,13 +2,20 @@ language: bash
 services: docker
 
 env:
-  image=eclecticiq/package
+  global:
+    image=eclecticiq/package
+  matrix:
+    - version=1.11.0 latest=true
+    - version=1.10.2
+    - version=1.10.1
+    - version=1.10.0
 
-script:
-  - travis_retry docker build --tag "$image:latest" .
-
-after_success:
-  - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
-    docker push $image;
-    fi
+jobs:
+  include:
+    - script: travis_retry docker build --build-arg FPM_VERSION=$version --tag "$image:$version" .
+    - script: docker tag "$image:$version" "$image:latest"
+      if: env(latest) = true
+    - script:
+        - docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
+          docker push $image
```

Comment:
shouldn't these be two different items in the list, or you need to add a semicolon or `&&`?

also I didn't know that `docker push $image` would push all tags, I hope that's the case!

Reply:
I think it does, we will see but after merging the PR.